lib/models/account-template.js

```diff
@@ -79,10 +79,20 @@ class AccountTemplate {
    * @return {Object}
    */
   static templateSubstitutionsFor (userAccount) {
+    let addOwlSameAs = userAccount.externalWebId &&
+           userAccount.tls_certificate_webid &&
+           userAccount.externalWebId !== userAccount.tls_certificate_webid
```

Comment:
Is this part correct.  Should it be camel case?

Reply:
Yes, it is a bug, I missed this rename, the fix has been commited